crates/api_common/src/community.rs

```diff
@@ -16,6 +16,64 @@ use serde_with::skip_serializing_none;
 #[cfg(feature = "full")]
 use ts_rs::TS;
 
+#[skip_serializing_none]
+#[derive(Debug, Serialize, Deserialize, Clone, Default)]
+#[cfg_attr(feature = "full", derive(TS))]
+#[cfg_attr(feature = "full", ts(export))]
+/// Create a tag for a community.
+pub struct CreateCommunityTag {
+  pub community_id: CommunityId,
+  pub id_slug: String,
+  pub name: String,
```

Comment:
What's this slug about? I see the tag table just has an id, ap_id, and a name.

Reply:
Probably to allow things like tag name with space, then slug has space replaced with underscore. But thats needlessly complicated and should be removed.

Reply:
name is the human-readable name. slug is the portion of the ap_id URL that is flexible. 

I added this for flexibility of choosing the immutable part of the URL. I'd expect frontends to usually auto-generate this based on the name, but have it be configurable for an advanced user.